source/patterns/@aws-solutions-constructs/aws-kinesisstreams-lambda/test/integ.existing.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ *  Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+// Imports
+import { KinesisStreamsToLambda, KinesisStreamsToLambdaProps } from '../lib';
+import { Stack, App, Aws } from '@aws-cdk/core';
+import * as kinesis from '@aws-cdk/aws-kinesis';
+import * as lambda from '@aws-cdk/aws-lambda';
+import * as iam from '@aws-cdk/aws-iam';
+
+// Setup
+const app = new App();
+const stack = new Stack(app, 'test-ks-lambda-stack');
+stack.templateOptions.description = 'Integration Test for aws-kinesisstreams-lambda';
+
+const lambdaRole = new iam.Role(stack, 'test-role', {
```

Comment:
There was no integration tests for bringing existing resources, so I created this one. 

I'm creating a role separately because the default one the CDK creates does not have permission to write to CloudWatch, and that'd cause a warning from cfn_nag (_"W58 Lambda functions require permission to write CloudWatch Logs"_)